libs/SalesforceSDK/src/com/salesforce/androidsdk/auth/HttpAccess.java

```diff
@@ -354,6 +360,15 @@ private HttpURLConnection createHttpUrlConnection(URI uri, String requestMethod)
     	if (uri != null) {
     		URL url = uri.toURL();
     		if (url != null) {
+
+    			/*
+    			 * FIXME: PATCH has been added to the latest OkHttp library,
```

Comment:
Something unrelated that I recently discovered. Made a note here, so I won't forget to fix it in the future.